packages/bot-web-ui/src/stores/transactions-store.ts

```diff
@@ -68,11 +68,38 @@ export default class TransactionsStore {
         );
         const statistics = trxs.reduce(
             (stats, { data }) => {
-                const { profit = 0, is_completed = false, payout = 0, buy_price = 0 } = data as TContractInfo;
+                const {
+                    profit = 0,
+                    is_completed = false,
+                    payout = 0,
+                    buy_price = 0,
+                    bid_price = 0,
+                } = data as TContractInfo;
                 if (is_completed) {
                     if (profit > 0) {
                         stats.won_contracts += 1;
                         stats.total_payout += payout;
+                    }
+                    if (bid_price > 0) {
+                        if (window.Blockly.derivWorkspace) {
```

Comment:
Since the user can make changes directly in the Blockly workspace I used this approach